templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+  <p>Are you really want to delete this Driver?</p>
```

Comment:
There is a grammatical error in the confirmation message. It should be 'Are you sure you want to delete this Driver?' instead of 'Are you really want to delete this Driver?'.